packages/calypso-data-stores/src/domain-suggestions/selectors.ts

```diff
@@ -6,7 +6,7 @@ import { select } from '@wordpress/data';
 /**
  * Internal dependencies
  */
-import { STORE_KEY } from '.';
+import { STORE_KEY } from './constants';
```

Comment:
What about naming this constant `DOMAIN_SUGGESTIONS_STORE_NAME`? Better that having `STORE_KEY` exports everywhere and needing to alias them to a more specific name all the time.

Reply:
These now live under their respective store exports, like `Verticals.STORE_KEY`.